packages/plugin-koa/src/request-info.js

```diff
@@ -0,0 +1,25 @@
+module.exports = ctx => {
+  if (!ctx) return {}
+  const request = ctx.req
+  const connection = request.connection
+  const address = connection && connection.address && connection.address()
+  const portNumber = address && address.port
+  const url = `${ctx.request.href}`
+  return {
+    url,
+    path: request.url,
+    httpMethod: request.method,
+    headers: request.headers,
+    httpVersion: request.httpVersion,
+    query: ctx.request.query,
+    connection: request.connection ? {
+      remoteAddress: request.connection.remoteAddress || request.ip,
+      remotePort: request.connection.remotePort,
+      bytesRead: request.connection.bytesRead,
+      bytesWritten: request.connection.bytesWritten,
```

Comment:
in the request handler is this most likely to be incorrect at the time of the error as we get the information before presumably a lot of the koa stack has been run?

Reply:
Good point, and it ties in with your other point abvout when we collect the data. Again this is how it was in bugsnag-node.